pom.xml

```diff
@@ -114,6 +114,7 @@
           <target>${jdk.version}</target>
           <encoding>${project.build.sourceEncoding}</encoding>
           <showWarnings>true</showWarnings>
+          <parameters>true</parameters>
```

Comment:
What does this do and why is it needed?

Reply:
By default, the java compiler substitutes the constructor's parameter's names with arg0, arg1, etc. With this option, the parameter's names are kept. 
For instance, given the following constructor:
```
public Foo(final String name, final BigInteger id) {
        this.name = name;
        this.id = id;
}
```
without that option will look like:
```
public Foo(final String arg0, final BigInteger arg1) {
        this.name = arg0;
        this.id = arg1;
}
```
witht that option enabled will look like:
```
public Foo(final String name, final BigInteger id) {
        this.name = name;
        this.id = id;
}
```
I've added it to parent pom as, this helps a lot the class injection from various bean mapping frameworks available on the web: such as BULL, Jackson, etc.
So coming back to your question, this is not strictly related to the Cobertura issue fix

Reply:
OK, I didn't know that but it sounds like a useful change.